### DISCUSSION TITLE
How to restart chia full node with a coomand?

### DISCUSSION THREAD

**Discussion by elmans1961** [Chia-Network/chia-blockchain#4597] [General]
I tried to restart my chia with a cmd but failed.No problem with starting chia for me but I cannot restart it.I tried chia -r ,chia node -r,chia --restart, chia restart .Commands chia start node and chia stop node works fine.But chia restart node(all,timeLord,etc) does not work with me.
Why I'm asking how to restart because connection to peer discinnects from time to time the last three days and I have to reload my PC to get connected.

**Comment by ghost** [comment]
Hi

What about stopping, then starting? For example:
1. chia stop -d all
2. chia start farmer

**Reply by elmans1961** [reply]
No problem.I messaged that I do it instead of restarting.But  there's a command restart.And in a help file it's marked as -r but does not work/

**Comment by ilyakooo0** [comment]
`chia start -r farmer`

**Reply by elmans1961** [reply]
Thank you!
